### DISCUSSION TITLE
Errno 2 No Such file or directory

### DISCUSSION THREAD

**Discussion by ItsXolten** [bmaltais/kohya_ss#2143] [Q&A]
C:\Users\syste\AppData\Local\Programs\Python\Python311\python.exe: can't open file 'Z:\\Program Files\\kohya_ss-master\\sd-scripts\\finetune\\make_captions.py': [Errno 2] No such file or directory 

This happens when trying to use BLIP Captioning

**Comment by dmaltsevQA** [comment]
Probably installation of BLIP weren't success. I'd tray WD14 and watch installation process
